--- debian/rules
+++ debian/rules
@@ -53,3 +53,9 @@
 install -m0755 -d debian/php-vitexsoftware-ease-core/usr/share/pkg-php-tools/autoloaders
 echo 'deb ease-core EaseCore/autoload.php' > debian/php-vitexsoftware-ease-core/usr/share/pkg-php-tools/autoloaders/php-vitexsoftware-ease-core
 
+override_dh_phpcomposer:
+mv composer.json composer.json.bak
+jq 'del(.require["composer-runtime-api"])' composer.json.bak > composer.json
+dh_phpcomposer
+mv composer.json.bak composer.json
+
